src/util/MultiMap.ts

```diff
@@ -0,0 +1,22 @@
+export class MultiMap<K, V> {
```

Comment:
This creates a new GC object around a GCed map. It seems that this could be avoided by extending `Map<K,V>` instead, which should work in AS, but might not "just work" in TS. Have you tried that?

Reply:
Unfortunately that won't work with `--target es5`. I can just add a `multiMapAdd` helper function though.